spec/crypto/decaf377/test_vectors.md

```diff
@@ -21,4 +21,19 @@ This table has hex-encodings of small multiples of the generator $B$:
 | $[12]B$ | `f43f4cefbe7326eaab1584722b1b4860de554b23a14490a03f3fd63a089add0b` |
 | $[13]B$ | `76c739a33ffd15cf6554a8e705dc573f26490b64de0c5bd4e4ac75ed5af8e60b` |
 | $[14]B$ | `200136952d18d3f6c70347032ba3fef4f60c240d706be2950b4f42f1a7087705` |
-| $[15]B$ | `bcb0f922df1c7aa9579394020187a2e19e2d8073452c6ab9b0c4b052aa50f505` |
\ No newline at end of file
+| $[15]B$ | `bcb0f922df1c7aa9579394020187a2e19e2d8073452c6ab9b0c4b052aa50f505` |
+
+## Group-to-hash
+
+This table has $(x,y)$ output affine Edwards coordinates after applying the Elligator map to the field element `2873166235834220037104482467644394559952202754715866736878534498814378075613`. Each subsequent test vector uses the s-coordinate of the previous result as input:
+
+| Affine x | Affine y |
+|-|-|
+| `1267955849280145133999011095767946180059440909377398529682813961428156596086` | `5356565093348124788258444273601808083900527100008973995409157974880178412098` |
+| `1502379126429822955521756759528876454108853047288874182661923263559139887582` | `7074060208122316523843780248565740332109149189893811936352820920606931717751` |
+| `2943006201157313879823661217587757631000260143892726691725524748591717287835` | `4988568968545687084099497807398918406354768651099165603393269329811556860241` |
+| `2893226299356126359042735859950249532894422276065676168505232431940642875576` | `5540423804567408742733533031617546054084724133604190833318816134173899774745` |
+| `2950911977149336430054248283274523588551527495862004038190631992225597951816` | `4487595759841081228081250163499667279979722963517149877172642608282938805393` |
+| `3318574188155535806336376903248065799756521242795466350457330678746659358665` | `7706453242502782485686954136003233626318476373744684895503194201695334921001` |
+| `3753408652523927772367064460787503971543824818235418436841486337042861871179` | `2820605049615187268236268737743168629279853653807906481532750947771625104256` |
+| `7803875556376973796629423752730968724982795310878526731231718944925551226171` | `7033839813997913565841973681083930410776455889380940679209912201081069572111` |
```

Comment:
Here it would probably be good to specify the s-coordinate, but it could be a later change than this PR.